src/Viewarea.js

```diff
@@ -160,6 +160,8 @@ x3dom.Viewarea = function(document, scene) {
     this.vrRightProjMatrix = new x3dom.fields.SFMatrix4f();
 
     this.vrControllerManager = new x3dom.VRControllerManager();
+    
+    this._inverseDevicePixelRatioInverse = 1.0 / window.devicePixelRatio;
```

Comment:
Is the double "inverse" in the variable name intended?

Reply:
thanks for the catch.